### ISSUE TITLE
Performance degradation from 168f66c

### ISSUE THREAD

**Issue by MohitSethi99** [EpicGames/raddebugger#119]
I use array of 1,000,000 MeshData (transform and vertex arrays) and I noticed huge performance difference regarding that (trying to evaluate s_Renderer3DData::Meshes) since the 168f66cc99455774e60be4e27e6368af99762fe0

Before the change:
https://github.com/EpicGames/raddebugger/assets/29519295/c196d370-d7d0-4cdf-8502-67a8acc8f54a

After the change
https://github.com/EpicGames/raddebugger/assets/29519295/d7fe3501-67af-4e1a-95eb-44a83a181b3c

I was able to reproduce it with this small program:
```cpp
struct mat4
{
	float v[16] = {0};
};

int main(int argc, const char** argv)
{
	auto ints = new std::array<mat4, 1'000'000>();
	delete ints;
	return 0;
}
```

**Comment by ryanfleury** [comment]
Oops, my bad! Will fix that ASAP.

**Comment by ryanfleury** [comment]
Fixed in 455dac958c948f838204891f024e4b08676b4cce.
